src/freebsd_sysctl.c

```diff
@@ -856,7 +909,11 @@ int do_system_ram(int update_every, usec_t dt) {
     (void)dt;
     static int mib_active_count[4] = {0, 0, 0, 0}, mib_inactive_count[4] = {0, 0, 0, 0}, mib_wire_count[4] = {0, 0, 0, 0},
                mib_cache_count[4] = {0, 0, 0, 0}, mib_vfs_bufspace[2] = {0, 0}, mib_free_count[4] = {0, 0, 0, 0};
+#if __FreeBSD_version < 1200029
```

Comment:
You can avoid these `#ifdef` statements with a `typedef`. I suggest to `typedef` the right one at the struct definition, to keep the code clean...